rktl_autonomy/scripts/train_rocket_league.py

```diff
@@ -15,13 +15,22 @@
 from os.path import expanduser
 import uuid
 
-if __name__ == '__main__':      # this is required due to forking processes
+"""
+All run env share a run_id
+This is where you specify launch argos: add to the env_kwargs
+Set the model that you will do to use for the training
+Log the training process for the specific run id
+Specify the stems and let the model learn wit these specific steps
+useful resource for stable_baselines3: https://stable-baselines3.readthedocs.io/_/downloads/en/master/pdf/
+"""
+
+if __name__ == '__main__':  # this is required due to forking processes
```

Comment:
I still find this comment pretty cryptic and not sure if it adds value to this file. We can add the stable baselines link to the wiki.

Reply:
I think that it was my own notes that I wrote that I forgot to delete. I left the baseline 3 resource in there for now, at the top of the file comment, so that it will not be lost.